docs/config-fcos-v1_5-exp.md

```diff
@@ -203,6 +203,7 @@ The Fedora CoreOS configuration is a YAML document conforming to the following s
     * **_threshold_** (int): sets the minimum number of pieces required to decrypt the device. Default is 1.
   * **_mirror_** (object): describes mirroring of the boot disk for fault tolerance.
     * **_devices_** (list of strings): the list of whole-disk devices (not partitions) to include in the disk array, referenced by their absolute path. At least two devices must be specified.
+* **extensions** (list of strings): A list of packages to layer on top of the OS.
```

Comment:
lowercase A.

Also, we should either a) add this to the OpenShift experimental spec docs, or (b) have the OpenShift spec validator explicitly reject the field.

Reply:
Is there a best practice on which to select? Specially if this could be changing?

Reply:
It's safer to disable the functionality in `openshift` and possibly enable it later, rather than the other way around.

Now that I think about it, `openshift` configs already have a way to specify OS extensions (`openshift.extensions`).  The last time we added that sort of duplication (for kernel arguments) we had `openshift` disable the fields inherited from the `fcos` variant, and continued to support only the `openshift`-specific fields there.

Reply:
This might tie into this thread: https://github.com/coreos/butane/pull/304#discussion_r790336682 we want this to be fcos + openshift thing. So it might require another chat with the MCO team. I am more leaning into creating a Issue that says to revisit this once it's time to stabilize the API. I think for now we need to allow both on openshift.

Reply:
Well, it also ties into https://github.com/coreos/butane/issues/300#issuecomment-1015041837.  Even if `openshift` inherits it now, that'll change when 4.11.0 is stabilized unless the MCO starts supporting Ignition spec 3.3.0.

So I'm okay leaving it enabled for now and filing a bug.  I've created stabilization milestones for the next `fcos` and `openshift` specs in the issue tracker, and we can attach to those.  Could you handle filing the issue?

Reply:
In that case, please do update the `openshift` spec docs as well.

Reply:
Opened: https://github.com/coreos/butane/issues/314